### PR TITLE
CGE-62

### PR DESCRIPTION
**Pull request by 21November**
Added redirect to index page when root address is entered.

### CODE REVIEW

File: src/main/java/com/hillel/items_exchange/config/MvcConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.core.Ordered;
+import org.springframework.web.servlet.config.annotation.*;
+import org.springframework.web.servlet.view.JstlView;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com")
+public class MvcConfiguration implements WebMvcConfigurer{
```
Comment (rpkyrych) [inline src/main/java/com/hillel/items_exchange/config/MvcConfiguration.java]: This class is excess - you should completely remove it.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -14,15 +14,16 @@ spring.datasource.password=
...
-spring.liquibase.user=root
-spring.liquibase.password=
 
 ### Hibernate Properties
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.ddl-auto=update
 
+## Spring View Resolver
+spring.mvc.view.prefix=html/
```
Comment (rpkyrych) [inline src/main/resources/application.properties]: I don't see the same configuration for test purpuses.

Comment (21November) [reply src/main/resources/application.properties]: Performed.

File: src/main/java/com/hillel/items_exchange/config/MvcConfiguration.java (hunk 1 above)
Comment (21November) [reply src/main/java/com/hillel/items_exchange/config/MvcConfiguration.java]: Performed.

**Pull request merged**